src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -656,8 +656,8 @@ shim::~shim()
     dev_fini();
 
     for (auto p : mCuMaps) {
```

Comment:
Given the changes to the changes to mCuMaps, I think this should be 
```suggestion
    for (const auto& p : mCuMaps) {
```

Reply:
Good eye. Fixed.